packages/vats/decentral-emerynet-vaults-config.json

```diff
@@ -0,0 +1,191 @@
+{
+  "$comment": "This SwingSet config file (see loadSwingsetConfigFile) is designed to bring up vaults in a decentralized networks in an automated fashion. It includes coreProposals to start vaults. Its testing facilities are limited to known testing keys for EC and Oracle operators.",
+  "bootstrap": "bootstrap",
+  "defaultReapInterval": 1000,
+  "coreProposals": [
+    "@agoric/vats/scripts/init-core.js",
+    {
+      "module": "@agoric/inter-protocol/scripts/init-core.js",
+      "entrypoint": "defaultProposalBuilder",
+      "args": [
+        {
+          "econCommitteeOptions": {
+            "committeeSize": 6
+          },
+          "endorsedUi": "NO ENDORSED UI",
+          "minInitialPoolLiquidity": "0"
+        }
+      ]
+    },
+    {
+      "module": "@agoric/inter-protocol/scripts/add-collateral-core.js",
+      "entrypoint": "defaultProposalBuilder",
+      "args": [
+        {
+          "debtLimitValue": 0,
+          "interchainAssetOptions": {
+            "denom": "ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA",
```

Comment:
I'm not sure what this denom should be, @arirubinstein

Reply:
we only have `ibc/toyusdc` and `ubld` in the faucet so it may be good to set vaults here with that?
or perhaps we can use `ubld`?

Reply:
`ubld` should work technically, though it's very much not want we want to in an economic sense. i better make sure product is aware / ok on this.

Reply:
`ubld` wouldn't work out-of-the-box either: it's already registered in the vbank, and this section would try to do that again and fail.

How about using `ibc/toyusd` for vaults _instead_ of PSM? 053c7dcad has details.

Reply:
Thanks to @michaelfig we have 9M tokens (with 6 decimals) to play with bearing the denom `ibc/06362C6F7F4FB702B94C13CD2E7C03DEC357683FD978936340B43FBFBC5351EB`